### DISCUSSION TITLE
starting date in return date calender

### DISCUSSION THREAD

**Discussion by nitGu** [dmuy/duDatepicker#51] [General]
i have two calender in flight booking web page if i select future date in 1st calender then 2nd calender should start with same date
i have stected in 1st calender

**Comment by dmuy** [comment]
Hi @nitGu,

This feature is not supported by default, you should make use of the configurations and events (ex. `setMinDate`, and the `dateChanged` event). But if the user is choosing a date range, you might want to consider using [date range mode](https://github.com/dmuy/duDatepicker/wiki/Usage#date-range)
